activeadmin.gemspec

```diff
@@ -22,10 +22,10 @@ Gem::Specification.new do |s|
   s.add_dependency 'arbre', '~> 1.1', '>= 1.1.1'
   s.add_dependency 'formtastic', '~> 3.1'
   s.add_dependency 'formtastic_i18n', '~> 0.4'
-  s.add_dependency 'inherited_resources', '~> 1.7'
+  s.add_dependency 'inherited_resources'
   s.add_dependency 'jquery-rails', '~> 4.2'
   s.add_dependency 'kaminari', '~> 1.0', '>= 1.0.1'
-  s.add_dependency 'railties', '>= 5.0', '< 5.3'
+  s.add_dependency 'railties', '>= 5.0'
```

Comment:
Let's keep an upper limit here. I think `< 6.0` to allow 6.0 prereleases works for now.

Reply:
I changed to keep the upper limit as `< 6.0`